packages/quantic/force-app/main/default/lwc/quanticSort/e2e/fixture.ts

```diff
@@ -0,0 +1,112 @@
+import {SortObject} from './sortObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+
+const pagerUrl = 's/quantic-sort';
```

Comment:
```suggestion
const pageUrl = 's/quantic-sort';
```

same here for consistency